tests/backup/backup_share_test.go

```diff
@@ -608,22 +609,23 @@ var _ = Describe("{ShareBackupWithUsersAndGroups}", func() {
 		Step("Share Backup with Full access to a user of View Only access group and Validate", func() {
 			log.InfoD("Share Backup with Full access to a user of View Only access group and Validate")
 			// Get user from the view access group
-			username, err := backup.GetRandomUserFromGroup(groups[0])
+			var err error
+			chosenUser, err = backup.GetRandomUserFromGroup(groups[0])
 			log.FailOnError(err, "Failed to get a random user from group [%s]", groups[0])
-			log.Infof("Sharing backup with user - %s", username)
+			log.Infof("Sharing backup with user - %s", chosenUser)
 
 			// Get Admin Context - needed to share backup and get backup UID
 			ctx, err := backup.GetAdminCtxFromSecret()
 			log.FailOnError(err, "Fetching px-central-admin ctx")
 
 			// Share backup with the user
 			backupName := backupNames[0]
-			err = ShareBackup(backupName, nil, []string{username}, FullAccess, ctx)
+			err = ShareBackup(backupName, nil, []string{chosenUser}, FullAccess, ctx)
 			log.FailOnError(err, "Failed to share backup %s", backupName)
 
 			// Get user context
-			ctxNonAdmin, err := backup.GetNonAdminCtx(username, "Password1")
-			log.FailOnError(err, "Fetching %s ctx", username)
+			ctxNonAdmin, err := backup.GetNonAdminCtx(chosenUser, "Password1")
```

Comment:
Would it be a better approach to use a global variable "password" instead of hard-coding it within the code?

Reply:
We need to do it for all tests. We can do it in a separate PR.